content/terms_and_definitions.md

```diff
@@ -1,285 +1,346 @@
 
-## external reference
+### select data type
 
-link to information outside the data set, with direct relevance to the specific information the link originates from inside the data set
+data type that enables a choice among several named data types
 
-## feature
+[SOURCE: ISO 10302-11:2004, 8.4.2, modified - Definition adapted from normative text]
 
-parametric information and additional property information modifying the shape representation of an element to which it applies
+## deprecation
 
-## group
+act of marking a schema construct to be deleted in a future major release of a specification
 
-collection of information that fulfills a specified purpose
+Note 1 to entry:  Complying interpreters shall still be able to import deprecated definitions.
 
-> NOTE A specialization of the general term object.
+Note 2 to entry:  Complying interpreters shall consider modifying export using the proposed alternative definitions instead of the deprecated ones.
 
-## identification
+## element
 
-capability to find, retrieve, report, change, or delete specific instances without ambiguity
+physical object with a stated function, form and position
 
-## instance
+## entity
 
-occurrence of an entity
+class of information defined by common properties
 
-> NOTE Similar to the term "instance of a class" in object oriented programming.
+[SOURCE: ISO 10303-11:2004, 3.3.6]
 
-## library
+## external reference
 
-catalogue, database or holder of data, that is relevant to information in the data set
+locator for referencing information stored outside of the populated schema
 
-> NOTE It is information referenced from an external source that is not copied into the data set.
+## exchange structure
 
-## model
+computer-interpretable format used for storing, accessing, transferring, and archiving data
 
-a data set, governed by the structure of an underlying schema, to meet certain data requirements
+[SOURCE: ISO 10303-1:2021, 3.1.36]
 
-> NOTE  Information models and building information models are examples for a model.
+## facility
+built asset
 
-> NOTE  In scope of this standard IFC models are populations of the IFC schema.
+physical structure, including the related site, serving one or more main purposes
 
-## model view definition
+[SOURCE: ISO 12911:2012, 3.9, modified - The words "related site works" have been replaced with "the related site". The admitted term "built asset" has been added.]
 
-Subset of a schema representing the data structure required to fulfill the data requirements within one or several exchange scenarios.
-Beside being a subset of a schema, a model view (or model view definition) also imposes additional constraints to the population of the subset schema. 
+## feature
 
-> EXAMPLE a mnodel view definition may define that an IfcBuilding always needs to have an IfcSite and an IfcProject.
+parametric information and additional property information modifying the shape representation of an element to which it applies
```

Comment:
I wouldn't use "shape representation" here because in way that remains unaltered as it is represented in the model.

```suggestion
conceptualization of certain design or manufacturing functionality to implicitly alter the geometric form of an element to be computed at import
```

I have an alternative for inspiration.